ui/components/app/snaps/snap-delineator/snap-delineator.js

```diff
@@ -83,7 +83,7 @@ export const SnapDelineator = ({
           />
           <Text
             variant={TextVariant.bodySm}
-            color={isError ? TextColor.textDefault : TextColor.textAlternative}
+            color={isError ? TextColor.errorDefault : TextColor.textAlternative}
```

Comment:
The issue asks for the color to be `textDefault`, perhaps always? @eriknson

Reply:
Visually, this would be the effect. Waiting on @eriknson for confirmation.

![Screenshot 2024-03-07 at 10 20 51](https://github.com/MetaMask/metamask-extension/assets/44679989/dcdf7068-825c-49f3-b882-f457c880e3fe)
![Screenshot 2024-03-07 at 10 19 48](https://github.com/MetaMask/metamask-extension/assets/44679989/0903310b-4251-4e66-b859-9fd900531d5e)

Reply:
I confirmed this with Erik